ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/SPGDPRManagePreferenceViewController.swift

```diff
@@ -12,13 +12,13 @@ class SPGDPRManagePreferenceViewController: SPNativeScreenViewController {
     struct Section {
         let header: SPNativeText?
         let contentConsent: [GDPRCategory]
-        let contentLegIntCategory: [GDPRCategory]?
+        let contentLegIntCategory: [GDPRCategory]
 
         init? (header: SPNativeText?, contentConsent: [GDPRCategory]?, contentLegIntCategory: [GDPRCategory]? = nil) {
             if contentConsent == nil || contentConsent?.isEmpty == true { return nil }
             self.header = header
             self.contentConsent = contentConsent! // swiftlint:disable:this force_unwrapping
```

Comment:
Is there a specific reason for `// swiftlint:disable:this force_unwrapping` being still here?

Reply:
Good catch, didn't notice that one.